backend/src/nodes/nodes/builtin_categories.py

```diff
@@ -0,0 +1,25 @@
+from .image import category as ImageCategory
+from .image_adjustment import category as ImageAdjustmentCategory
+from .image_filter import category as ImageFilterCategory
+from .image_dimension import category as ImageDimensionCategory
+from .image_channel import category as ImageChannelCategory
+from .image_utility import category as ImageUtilityCategory
+from .utility import category as UtilityCategory
+from .pytorch import category as PyTorchCategory
+from .ncnn import category as NCNNCategory
+from .onnx import category as ONNXCategory
+
+
+builtin_categories = [
```

Comment:
Should this file really be in `src/nodes/nodes`? This folder only contains node definitions otherwise. It feels weird that this file is in there as well.

Reply:
It also contains the category definitions though